pages/science/reactors/cvodes.rst

```diff
@@ -0,0 +1,244 @@
+and the maximum time step. A reduction of the latter value is particularly useful 
+when dealing with abrupt changes in the boundary conditions (for example, 
+opening/closing valves; see also the `IC engine example </examples/python/reactors
+/ic_engine.py.html>`__).
+
+How Does Cantera's Reactor Network Time Integration Feature Actually Work?
+==========================================================================
+
+There's a great description of the science behind Cantera's reactor network 
+simulation capabilities available on the Cantera website, 
+`here <https://cantera.org/science/reactors/reactors.html>`__. This section will go into more 
+developer-oriented detail about how the last step, ``ReactorNet``'s 
+`time integration methods <https://cantera.org/science/reactors/reactors.html#time-
+integration-for-reactor-networks>`__, actually work. A ``ReactorNet`` object doesn't 
+perform time integration on its own. Cantera generates a Jacobian array for the set 
+of ``Reactor`` objects contained in the ``ReactorNet``. This Jacobian, along with 
+functions to evaluate the residual of each ODE representing the system, is passed to 
+an ``Integrator``.
+
+Following integration from Reactor Network creation to completion
+-----------------------------------------------------------------
+
+Step 1: Reactor(s) created in Reactor Network
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+First, let's take a look at a basic example to see how we might utilize Cantera's time integration 
+functionality. We'll simulate an isolated reactor in Python that is homogeneously filled by a gas 
+mixture (the gas state used in this example is arbitrary, but interesting because it's 
+explosive). Then we'll advance the simulation in time to an (arbitrary) absolute time of 
+1 second, noting the changes in the state of the gas.
+
+.. code-block::
+
+    >>> import cantera as ct                           #import Cantera's Python module
+    >>> gas = ct.Solution("gri30.yaml")                #create a default GRI-Mech 3.0 gas mixture
+    >>> gas.TPX = 1000.0, ct.one_atm, "H2:2,O2:1,N2:4" #set gas to an interesting state
+    >>> reac = ct.IdealGasReactor(gas)                 #create a reactor containing the gas
+    >>> sim = ct.ReactorNet([reac])                    #add the reactor to a new ReactorNet simulator
+    >>> gas()                #view the initial state of the mixture (state summary will be printed to console)
+    >>> sim.advance(1)       #advance the simulation to the specified absolute time, t = 1 sec
+    >>> gas()                #view the updated state of the mixture, reflecting properties at t = 1 sec
+
+For a more advanced example that adds inlets and outlets to the reactor, see Cantera's combustor example 
+(`Python <https://github.com/Cantera/cantera/blob/main/interfaces/cython/cantera/examples/reactors/combustor.py>`__ 
+| `C++ <https://github.com/Cantera/cantera/blob/main/samples/cxx/combustor/combustor.cpp>`__). Additional examples 
```

Comment:
Better to link to these directly on the Cantera website as `/examples/python/reactors/combustor.py.html` and `/examples/cxx/combustor.html`.